ppanero/docker-services
# env: support specific versions and latest as string

Tested: 

- export ES_VERSION=ES_6_LATEST
- export ES_VERSION=ES_7_LATEST
- export ES_VERSION=6.6.0
- export ES_VERSION=ES_9_LATEST (FAILS, outputting a message)

Works as expected.
